.github/workflows/go.yml

```diff
@@ -25,8 +24,7 @@ jobs:
     - name: Get dependencies
       run: |
         go mod download
-        go get -u honnef.co/go/tools/cmd/staticcheck
-        curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.27.0
+        curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.39.0
```

Comment:
Highly recommend this action https://github.com/golangci/golangci-lint-action which also outputs errors in pr diffs etc.